hokusai/commands/run.py

```diff
@@ -39,10 +33,10 @@ def run(context, command, tty, tag, env):
         "containers": [
           {
             "args": command.split(' '),
-            "name": job_name,
+            "name": name,
             "image": image_name,
             "imagePullPolicy": "Always",
-            "env": environment,
+            'envFrom': [{'configMapRef': {'name': "%s-environment" % config.project_name}}],
```

Comment:
Does this mean the `.yml` files generated by `hokusai setup` will already include the `envFrom` directive? 👍